src/modules/auth/controller.js

```diff
@@ -28,6 +29,28 @@ const tryAuth = async (req, res, next) => {
   }
 };
 
+const refreshToken = async (req, res, next) => {
+  try {
+    const tokenPair = authService.generateTokenPair({
+      id: req.user.id,
+      email: req.user.email,
+    });
+
+    const bearerToken = authService.processTokenPair(res, tokenPair);
+
+    return res.sendResponse(StatusCodes.OK, { bearerToken });
```

Comment:
Different naming of the token field
You already named token as `accessToken` so keep using that name